n26/api.py

```diff
@@ -440,9 +445,15 @@ def _request_mfa_approval(mfa_token: str):
     def _complete_authentication_flow(self, mfa_token: str) -> dict:
         LOGGER.debug("Completing authentication flow for mfa_token {}".format(mfa_token))
         mfa_response_data = {
-            "grant_type": "mfa_oob",
             "mfaToken": mfa_token
         }
+
+        if self.config.mfa_type == "otp":
+            mfa_response_data['grant_type'] = "mfa_otp"
+            mfa_response_data['otp'] = input('Enter the 6 digit SMS OTP code: ')
```

Comment:
Hmm this is a hard one... I would like to separate the API from user interaction to allow devs using this library elsewhere to be able to provide other means of user interaction apart from console input. But I think we can do this in a separate Issue/PR.

But since we use `click` for everything terminal related pleas use their built in `prompt` function to request user input:
https://click.palletsprojects.com/en/7.x/prompts/